src/main/java/com/backbase/ct/bbfuel/input/TransactionsReader.java

```diff
@@ -25,8 +25,19 @@ public TransactionsPostRequestBody loadSingle(String externalArrangementId) {
         return getRandomFromList(load(globalProperties.getString(CommonConstants.PROPERTY_TRANSACTIONS_DATA_JSON)))
                 .withId(UUID.randomUUID().toString())
                 .withArrangementId(externalArrangementId)
-                .withBookingDate(DateUtils.addDays(new Date(), generateRandomNumberInRange(-180, 0)))
-                .withCheckSerialNumber(new BigDecimal(generateRandomNumberInRange(1, 99999999)));
+                .withBookingDate(DateUtils.addDays(new Date(), generateRandomNumberInRange(-180, 0)));
+    }
+
+    /**
+     * To be able to retrieve check images for a transaction, we need at least one of the transactions with
+     * with transactions id = TRANS0000000000001 or TRANS0000000000002 in transactions list.
+     * We need to set the date of the transaction to today so that we can easily find that transaction and test it.
+     */
+    public static TransactionsPostRequestBody loadSingleWithCheckImages(String externalArrangementId) {
+        TransactionsPostRequestBody defaultBody = this.loadSingle(externalArrangementId);
```

Comment:
`defaultBody` is not being used anymore.

Reply:
yes good catch, fixed now.